bootstrap/src/test.rs

```diff
@@ -106,6 +112,19 @@ impl TestCase {
         log::debug!("running {:?}", command);
         command.status().unwrap();
     }
+
+    pub fn build_lib(&self, manifest: &Manifest) {
+        std::fs::create_dir_all(self.output.parent().unwrap()).unwrap();
```

Comment:
`let output_dir = self.output.parent().unwrap()` since it is used twice. But why do you need to navigate to the parent directory for this?

Reply:
@Wybxc could you clarify why this needs to change to the parent directory of `output`? That doesn't seem to make sense.